### PR TITLE
ReincubateLtd.iPhoneBackupExtractorSetup version 7.3.5.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98709)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @upintheairsheep,

The installation isn't unattended:
image

Is there an installer switch to bypass this license agreement and have it install automatically?

**Comment by upintheairsheep** [comment]
> Hi @upintheairsheep,
> 
> The installation isn't unattended: image
> 
> Is there an installer switch to bypass this license agreement and have it install automatically?

Try checking the type of installer.

File: manifests/r/ReincubateLtd/iPhoneBackupExtractorSetup/7.3.5.0/ReincubateLtd.iPhoneBackupExtractorSetup.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: ReincubateLtd.iPhoneBackupExtractorSetup
+PackageVersion: 7.3.5.0
+Installers:
+- InstallerUrl: https://reincubate.com/res/labs/iphonebe/iphonebackupextractor-latest.exe
+  Architecture: neutral
+  InstallerType: exe
+  InstallerSha256: 57A16B2DD478BB659947E36067D1472197DBBF009B7F2BC681D7F08450510366
+  InstallerSwitches:
+    Silent: /quiet
+    SilentWithProgress: /passive
```
Comment (stephengillie) [inline manifests/r/ReincubateLtd/iPhoneBackupExtractorSetup/7.3.5.0/ReincubateLtd.iPhoneBackupExtractorSetup.installer.yaml:13]:
```suggestion
    Silent: /S
    SilentWithProgress: /S
```

Comment (stephengillie) [reply manifests/r/ReincubateLtd/iPhoneBackupExtractorSetup/7.3.5.0/ReincubateLtd.iPhoneBackupExtractorSetup.installer.yaml:13]: Despite appearing to be a Windows Installer package, this responds to Nullsoft switches.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**